docs/conf.py

```diff
@@ -50,9 +51,25 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme = "sphinx_rtd_theme"
+pycytominer_hex = "#c24dd6"
```

Comment:
Consider using a color which will create greater contrast here to help with [web accessibility](https://webaim.org/resources/contrastchecker/?fcolor=C24DD6&bcolor=F8F9FB).

Reply:
thanks for pointing to that website! it is very helpful. I've darkened the purple and will add in next series of commits

Reply:
fixed in 787b20b25a3f122078aee982c13dfbc1dd5331ed